code-challenges.test.js

```diff
@@ -22,9 +22,41 @@ const secretCodeWord2 = "Gobbledygook"
 const secretCodeWord3 = "Eccentric"
 // Expected output: "3cc3ntr1c"
 
+describe("theSecret", () => {
+    it("takes in a string and returns a coded message. The coded message converts 'a' to 4, 'e' to 3, 'i' to 1, and 'o' to 0.", () => {
+      const secretCodeWord1 = ["Lackadaisical"]
+      const secretCodeWord2 = ["Gobbledygook"]
+      const secretCodeWord3 = ["Eccentric"]
+      expect(theSecret(secretCodeWord1)).toEqual(expect.arrayContaining(["L4ck4d41s1c4l"]))
+      expect(theSecret(secretCodeWord2)).toEqual(expect.arrayContaining(["Gobbledygook"]))
+      expect(theSecret(secretCodeWord3)).toEqual(expect.arrayContaining(["Eccentric"]))
+    })
+  })
+
+// Fill in "describ/ it / expect statment"
+// Run test
+// ReferenceError: theSecret is not defined
 
 // b) Create the function that makes the test pass.
 
+// Write function that takes in specific letter and if there is that letter reuturn this other specific number.
+
+const theShuffler = (string) => {
+  return string.filter(value => {
+      if(value a){
+          return 4
+      }
+      if(value e){
+        return 3
+    }
+    if(value i){
+        return 1
+    }
+    if(value o){
+        return 0
+  })
+}
```

Comment:
I suspect this did not work. What kind of results did you get from this code? What did you do to chase the rabbit down the hole?